pkg/controllers/metrics/nodes.go

```diff
@@ -174,7 +152,7 @@ func publishNodeCountsForProvisioner(provisioner string, consumeNodesWith consum
 			})))
 		}
 
-		for _, os := range osValues {
+		for os := range osValues {
```

Comment:
https://github.com/awslabs/karpenter/pull/780 removes most things related to OS since we currently only support linux. You can remove OS metrics here for now and we'll defer until we implement multiple OS support.

https://github.com/awslabs/karpenter/pull/780/files#diff-1a039ebbb2e08a2c87ccc7228041496010992b28a47f2e7e13a9840eee5fc0f4L59